internal/runners/manifest/manifest.go

```diff
@@ -0,0 +1,155 @@
+	"github.com/ActiveState/cli/pkg/platform/runtime/target"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type primeable interface {
+	Output() output.Outputer
+	Project() *project.Project
+	Auth() *authentication.Auth
+	Analytics() analytics.Dispatcher
+	SvcModel() *model.SvcModel
+	Config() *config.Instance
+}
+
+type Manifest struct {
+	out       output.Outputer
+	project   *project.Project
+	auth      *authentication.Auth
+	analytics analytics.Dispatcher
+	svcModel  *model.SvcModel
+	cfg       *config.Instance
+}
+
+func NewManifest(prime primeable) *Manifest {
+	return &Manifest{
+		out:       prime.Output(),
+		project:   prime.Project(),
+		auth:      prime.Auth(),
+		analytics: prime.Analytics(),
+		svcModel:  prime.SvcModel(),
+		cfg:       prime.Config(),
+	}
+}
+
+func (m *Manifest) Run() (rerr error) {
+	if m.project == nil {
+		return rationalize.ErrNoProject
+	}
+
+	m.out.Notice(locale.Tl("manifest_operating_on_project", "Operating on project: [ACTIONABLE]{{.V0}}[/RESET], located at [ACTIONABLE]{{.V1}}[/RESET]\n", m.project.Namespace().String(), m.project.Dir()))
+
+	reqs, err := m.fetchRequirements()
+	if err != nil {
+		return errs.Wrap(err, "Could not fetch requirements")
+	}
+
+	artifacts, err := m.fetchArtifacts()
+	if err != nil {
+		return errs.Wrap(err, "Could not fetch artifacts")
+	}
+
+	vulns, err := m.fetchVulnerabilities(reqs)
+	if err != nil {
+		return errs.Wrap(err, "Could not fetch vulnerabilities")
+	}
+
+	m.out.Print(newRequirements(reqs, artifacts, vulns))
+
+	if len(vulns) > 0 {
+		m.out.Notice(locale.Tl("manifest_vulnerabilities_info", "\nFor CVE info run '[ACTIONABLE]state security[/RESET]'"))
+	}
+
+	return nil
+}
+
+func (m *Manifest) fetchRequirements() ([]bpModel.Requirement, error) {
+	commitID, err := localcommit.Get(m.project.Dir())
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not get commit ID")
+	}
+
+	bp := model.NewBuildPlannerModel(m.auth)
+	expr, _, err := bp.GetBuildExpressionAndTime(commitID.String())
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not get remote build expr and time")
+	}
+
+	reqs, err := expr.Requirements()
+	if err != nil {
+		return nil, errs.Wrap(err, "Could not get requirements")
+	}
+
+	return reqs, nil
+}
+
+func (m *Manifest) fetchArtifacts() ([]*artifact.Artifact, error) {
+	if strings.EqualFold(os.Getenv(constants.DisableRuntime), "true") {
+		return nil, nil
+	}
+
+	target := target.NewProjectTarget(m.project, nil, target.TriggerManifest)
+	rt, err := runtime.New(target, m.analytics, m.svcModel, m.auth, m.cfg, m.out)
+	if err != nil {
+		return nil, locale.WrapError(err, "err_packages_update_runtime_init", "Could not initialize runtime.")
+	}
+
+	if rt.NeedsUpdate() {
+		m.out.Notice(locale.T("manifest_runtime_needs_update"))
+	}
+
+	return rt.ResolvedArtifacts()
+}
+
+func (m *Manifest) fetchVulnerabilities(reqs []bpModel.Requirement) (vulnerabilities, error) {
+	if !m.auth.Authenticated() {
+		return nil, nil
```

Comment:
This effectively means we will say "None Detected" under CVEs if you are not authenticated. Instead of "None Detected" we should print "Authenticate to View" in this scenario.